style.css

```diff
@@ -0,0 +1,977 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  max-width: 1500px;
+}
+
+.hide-box,
+.right-nav,
+.hidden {
+  display: none;
+}
+
+.mobile ul {
+  display: flex;
+  justify-content: flex-end;
+  list-style-type: none;
+  padding: 8px;
+}
+
+.nav-search {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.fa-search,
+.fa-bars {
+  border-left: 1px solid rgb(210, 210, 210);
+  padding: 9px;
+  height: 45px;
+}
+
+.logo {
+  height: 48px;
+  width: 106px;
+  position: relative;
+  bottom: 45px;
+}
+
+.fa-search {
+  color: #aaa;
+  font-size: 23px;
+}
+
+.fa-bars {
+  color: #f42;
+  font-size: 24px;
+}
+
+header {
+  position: sticky;
+  top: 0;
+  left: 0;
+  background-color: rgb(250, 250, 250);
+  z-index: 1;
+  height: 48px;
+  margin-bottom: 10px;
+}
+
+.headings {
+  padding: 14px 12px;
+  position: relative;
+  bottom: 40px;
+}
+
+.main_image {
+  display: block;
+}
+
+.main_image-one {
+  background:
+    linear-gradient(rgba(173, 173, 173, 0.2), rgba(20, 20, 20, 1.3)),
+    url("images/nitendo.png");
+  background-repeat: no-repeat;
+  height: 40vh;
+  margin-bottom: 10px;
+  margin-left: 8px;
+  margin-right: 8px;
+}
+
+.main_image-one h5,
+.main_image-three h5 {
+  color: #fff;
+  position: relative;
+  top: 80px;
+  left: 10px;
+}
+
+.main_image-two h2 {
+  color: #fff;
+  position: relative;
+  top: 120px;
+  left: 10px;
+}
+
+.main_image-one h2 {
+  color: #fff;
+  position: relative;
+  left: 10px;
+  top: 80px;
+  font-size: 26px;
+}
+
+.main_image-three h2 {
+  color: #fff;
+  position: relative;
+  left: 10px;
+  top: 80px;
+}
+
+.details1 {
+  color: #aaa;
+  position: relative;
+  left: 10px;
+  top: 80px;
+}
+
+.details2 {
+  color: #aaa;
+  position: relative;
+  top: 130px;
+  left: 10px;
+}
+
+.details3 {
+  color: #aaa;
+  position: relative;
+  left: 10px;
+  top: 80px;
+}
+
+.main_image-two {
+  background:
+    linear-gradient(rgba(173, 173, 173, 0.2), rgba(20, 20, 20, 1)),
+    url("images/usb.jpg");
+  background-repeat: no-repeat;
+  height: 40vh;
```

Comment:
Here is how it looks side to side, just use 30 vh for your 3 elements and that should fix it (vh is the % of the screen you are using, if you put 40vh for each element, that will be 120% that's why your images are bigger:
![image](https://user-images.githubusercontent.com/53256139/87946574-cf80fa00-ca67-11ea-9146-37840da9a778.png)